src/app/examples/si-navbar/si-navbar-primary-custom-content.html

```diff
@@ -4,7 +4,7 @@
   [primaryItems]="primaryItems"
   [account]="{
     title: 'Jane Smith',
-    email: 'jane.smith.simpl@siemens.com',
+    email: 'jane.smith@siemens.com',
```

Comment:
We still should have prefix here:
```suggestion
    email: 'jane.smith.element@siemens.com',
```